src/main/java/org/gbif/api/vocabulary/License.java

```diff
@@ -0,0 +1,84 @@
+package org.gbif.api.vocabulary;
+
+import com.google.common.base.Strings;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
I found a convenient method in commons-lang3 (an existing dependency in gbif-api) to accomplish what I wanted:

 StringUtils.removeEnd(licenseUrl, "/");